src/customer/strategies/googlepay/googlepay-customer-strategy.ts

```diff
@@ -139,7 +139,9 @@ export default class GooglePayCustomerStrategy implements CustomerStrategy {
     }
 
     private _onPaymentSelectComplete(): void {
-        this._formPoster.postForm('/checkout.php', {
+        const storeUrl = window.location.pathname === '/embedded-checkout' ? `${window.location.ancestorOrigins[0]}/checkout` : 'checkout.php';
```

Comment:
Do you need to support Firefox? I don't think `ancestorOrigins` is supported in Firefox. Also, what data are you actually posting? Are you just trying to do a redirect?

Reply:
Hey @davidchin, according to the conversation we had on slack, I made the suggested change and it works as expected indeed as we want to redirect to the WordPress store URL instead of the default store, please let me know any feedback